travel-event-host-app/src/components/searchSection/searchEventsFilterBox/filterBox.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+import { useEffect, useState } from 'react';
+import {
+  Box,
+  MenuItem,
+  Select,
+  Checkbox,
+  FormGroup,
+  FormControlLabel,
+  Typography,
+  IconButton,
+} from '@mui/material';
+import categoriesArr from '@/lib/categoryArray';
+import CategoriesType from '@/components/searchSection/types';
+import styles from '../styles.module.css';
+import CloseIcon from '@mui/icons-material/Close';
+
+const updateCategories = () => {
+  let updateCategories: { [key: string]: boolean } = {};
+
+  Object.values(categoriesArr).map((category) => (updateCategories[category] = false));
+  return updateCategories;
+};
+
+const FilterBox = ({
+  setCategories,
+  categories,
```

Comment:
We should probably rename this prop to be more descriptive of what it is doing (keeping track of the checked state of the categories)
how about `categoryCheckboxStates` ?